create_graphics.py

```diff
@@ -375,8 +376,9 @@ def parallel_maps(cla, fhr, ds, level, model, spec, variable, workdir,
 
     # Build the output path
     png_suffix = level if level != 'ua' else ''
-    tile_label = f'_{tile}' if tile != 'full' else ''
+    tile_label = f'_{tile}_' # if tile != 'full' else ''
```

Comment:
I don't think we need this line anymore. Consider just using "_{tile}_" in line 381 directly.